app/src/main/java/de/othr/plantico/database/daos/OwnedPlantDAO.kt

```diff
@@ -0,0 +1,20 @@
+package de.othr.plantico.database.daos
+
+import androidx.room.Dao
+import androidx.room.Insert
+import androidx.room.OnConflictStrategy
+import androidx.room.Query
+import de.othr.plantico.database.entities.OwnedPlant
+import kotlinx.coroutines.flow.Flow
+
+@Dao
+interface OwnedPlantDAO {
+    @Query("SELECT * FROM ownedplant ORDER BY id")
+    fun getAllOwnedPlants(): Flow<List<OwnedPlant>>
+
+    @Insert(onConflict = OnConflictStrategy.IGNORE)
```

Comment:
Is there an error when insert failt? Because it seems like you ignore it.

Reply:
When the insert fails, it just is ignored, but the transaction still continues, see here: https://stackoverflow.com/questions/65808302/android-room-database-difference-between-onconflictstrategy-abort-vs-ignore